09a_length/MySite/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> MySite </title>
+        <link rel="stylesheet" href="style.css"/>
+    </head>
+    <body>
+    <img src="http://when-will.net/images/artikel/2014/december/Tokyo-Ghoul.jpg"></img>
```

Comment:
whoops: `img` tags don't need to have a closing tag: `<img src="..." />` should suffice :-)